xbmc/addons/include/xbmc_pvr_types.h

```diff
@@ -75,10 +75,10 @@ struct DemuxPacket;
 #define PVR_STREAM_MAX_STREAMS 20
 
 /* current PVR API version */
-#define XBMC_PVR_API_VERSION "1.9.2"
+#define XBMC_PVR_API_VERSION "1.9.3"
```

Comment:
The current version has been bumped to `1.9.3` in the mean time, you'll have to change this to `1.9.4`.